boards/arm/nucleo_l4r5zi/doc/nucleol4r5zi.rst

```diff
@@ -0,0 +1,252 @@
+.. _nucleo_l4r5zi_board:
+
+ST Nucleo L4R5ZI
+################
+
+Overview
+********
+
+The Nucleo L4R5ZI board features an ARM Cortex-M4 based STM32L4R5ZI MCU
+with a wide range of connectivity support and configurations. Here are
+some highlights of the Nucleo L4R5ZI board:
+
+
+- STM32 microcontroller in LQFP144 package
+- Two types of extension resources:
+  - Arduino Uno V3 connectivity
+  - ST morpho extension pin headers for full access to all STM32 I/Os
+- On-board ST-LINK/V2-1 debugger/programmer with SWD connector
+- Flexible board power supply:
+  - USB VBUS or external source(3.3V, 5V, 7 - 12V)
+  - Power management access point
+- Three User LEDs: LD1 (Green), LD2 (Blue), LD3 (Red)
+- Two push-buttons: USER and RESET
+
+.. image:: img/nucleo144_ulp_logo_1024.jpg
+     :width: 250px
+     :align: center
+     :height: 250px
+     :alt: Nucleo L4R5ZI
+
+More information about the board can be found at the `Nucleo L4R5ZI website`_.
+
+Hardware
+********
+
+The STM32L4R5ZI SoC provides the following hardware IPs:
+
+- Ultra-low-power with FlexPowerControl (down to 130 nA Standby mode and 100 uA/MHz run mode)
```

Comment:
keep line length < 80 chars please (unless unavoidable)

Not sure what "100 uA/MHz" means, if it's "100 uA and 100 MHz" it would be better to write it out as such (without a / )